geoalchemy2/elements.py

```diff
@@ -113,6 +113,35 @@ def desc(self):
         return binascii.hexlify(self.data)
 
 
+class EWKBElement(_SpatialElement, functions.Function):
+    """
+    Instances of this class wrap a EWKB value. Geometry values read
+    from the database are converted to instances of this type. You may
+    need to create instances of ``EWKBElement`` yourself.
+
+    Note: you cannot create ``EWKBElement`` objects directly from Shapely
+    geometries using the :func:`geoalchemy2.shape.from_shape` function as there
+    is currently no support in Shapely for SRIDs. Instead, you must have a
+    valid EWKB (e.g. beggining with SRID=4326;) and use directly
+    a ``EWKBElement`` (e.g. ``EWKB(buffer(ewkb))``).
```

Comment:
Also, isn't this entire note in opposition to your changes to `from_shape`? It looks to me that with your changes the `from_shape` function does support creating `EWKBElement`. No?